cleanup/apt-cleanup.service

```diff
@@ -12,3 +12,7 @@ ExecStart=/bin/bash -c "apt-get purge -y $(dpkg --list | egrep 'linux-(image|ima
 
 # Purge removed packages
 ExecStart=/bin/bash -c "apt-get autoremove -y; apt-get purge -y $(dpkg --list | grep '^rc' | awk '{print $2}') | tr '[:space:]' ' '"
+
+# Clears out the local repository of retrieved package files. It removes everything but the lock
+# file from /var/cache/apt/archives/ and /var/cache/apt/archives/partial/. (man apt-get)
+ExecStart=/bin/bash -c "apt-get clean -y"
```

Comment:
This should be not necessary anymore, right?

Reply:
The difference is that `now.sls` only is run manually, while this is run periodically - on top of the APT setting. So it would be duplicate and running every time

Reply:
It is supposed to run every time. It is not an heavy command and should not be a problem. Also this will clean the directories on already existing minions without running `now.sls`. 

It is also possible that somebody overrides the APT settings and save packages in the cache.